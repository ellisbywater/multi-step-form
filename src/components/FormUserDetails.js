import React, {Component} from 'react'

class FormUserDetails extends Component {
    continue = e => {
        e.preventDefault()
    }
    render() {
        const { values, handleChange } = this.props
        return (
            <div>
                Form User Details
            </div>
        )
    }
}

export default FormUserDetails