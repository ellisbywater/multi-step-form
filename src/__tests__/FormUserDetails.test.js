import React from 'react'
import { shallow } from 'enzyme'

import FormUserDetails from '../components/FormUserDetails'

describe('FormUserDetails', () => {
    it('should render correctly', () => {
        const component = shallow(<FormUserDetails />)
        expect(component).toMatchSnapshot()
    })
})
