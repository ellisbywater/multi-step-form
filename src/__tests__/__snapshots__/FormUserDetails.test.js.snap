// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormUserDetails should render correctly 1`] = `ShallowWrapper {}`;
